CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.4.0
+  - added `isolation_level` to override `ConsumerConfig.ISOLATION_LEVEL_CONFIG` to allow for `read_committed` [#44](https://github.com/logstash-plugins/logstash-integration-kafka/pull/44)
```

Comment:
too much detail on the internals, users might not care about: `ConsumerConfig.ISOLATION_LEVEL_CONFIG`
```suggestion
 - added the input `isolation_level` to allow fine control of whether to return transactional messages [#44](https://github.com/logstash-plugins/logstash-integration-kafka/pull/44)
```